### PR TITLE
Make Linux sandbox error message more useful

### PR DESCRIPTION
**Pull request by diracdeltas**
Fix https://github.com/brave/browser-laptop/issues/7210
Address https://github.com/brave/browser-laptop/issues/6902#issuecomment-285927831

### CODE REVIEW

File: patches/v8/filter.patch [hunk 1]
```diff
@@ -57,7 +57,7 @@ index 8587356ddcca34cdcad716ba7395666ec7ce4380..ad3e31e8ae859c0b02a57b6d2867ae4a
    }
  }
 diff --git a/src/inspector/inspector.gyp b/src/inspector/inspector.gyp
-index 6246f0ac0e7edfbe2272a577a648dcf08f868bb1..8dd41a110f63cb25f7bf3abfa9b42a3a910e78e0 100644
+index c70722f852b60831250b9c91bdf3abf71650425d..9a001e756ef61565cd144da12e63b49964693152 100644
```
Comment (bridiver) [inline patches/v8/filter.patch]: this doesn't look like it is supposed to be part of this change

Comment (diracdeltas) [inline patches/v8/filter.patch]: it was generated by `npm run update_patches`. running this command on muon/master with no chromium changes produces the following diff:

```
diff --git a/patches/v8/filter.patch b/patches/v8/filter.patch
index 066532349..aa5fb63fe 100644
--- a/patches/v8/filter.patch
+++ b/patches/v8/filter.patch
@@ -57,7 +57,7 @@ index 8587356ddcca34cdcad716ba7395666ec7ce4380..ad3e31e8ae859c0b02a57b6d2867ae4a
    }
  }
 diff --git a/src/inspector/inspector.gyp b/src/inspector/inspector.gyp
-index 6246f0ac0e7edfbe2272a577a648dcf08f868bb1..8dd41a110f63cb25f7bf3abfa9b42a3a910e78e0 100644
+index c70722f852b60831250b9c91bdf3abf71650425d..9a001e756ef61565cd144da12e63b49964693152 100644
 --- a/src/inspector/inspector.gyp
 +++ b/src/inspector/inspector.gyp
 @@ -4,7 +4,7 @@
```

Comment (bridiver) [inline patches/v8/filter.patch]: update_patches will pull in whatever you have locally, but this clearly isn't related to your changes so safer to just remove it I think

**Comment by diracdeltas** [comment]
removed filter.patch patch

**Comment by bridiver** [comment]
++

**Pull request merged**